bulk-model-sync-lib-mps-test/src/test/kotlin/org/modelix/model/sync/bulk/lib/test/ModelImporterWithMPSAdaptersTest.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2024.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.modelix.model.sync.bulk.lib.test
+
+import jetbrains.mps.persistence.DefaultModelRoot
+import org.jetbrains.mps.openapi.language.SConcept
+import org.modelix.model.data.ModelData
+import org.modelix.model.mpsadapters.MPSModuleAsNode
+import org.modelix.model.sync.bulk.ModelImporter
+import org.modelix.model.sync.bulk.asExported
+
+class ModelImporterWithMPSAdaptersTest : MPSTestBase() {
+
+    fun testRootInDifferentOrderCanBeImported_with_oneEmptyModule() = runWrite {
```

Comment:
Cant' we use the multi-word backticks notation here, too?

Reply:
Sure. Done.
I also update the pattern on how to specify the test data.